### PR TITLE
Fix #3 README.md

### LINKED ISSUE #3: Set up README.md

### PR DESCRIPTION
**Pull request by jsoderholm**
Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,131 @@
...
+### Tobias Hansson
+
+- Lorem Ipsum
+
+### John Söderholm
+
+- Lorem Ipsum
+
+### Tianxing Wu
+
```
Comment (TerenceNg03) [inline README.md:130]: - Implement web request handling
- Implement post commit status
- Set up project

File: README.md [hunk 2]
```diff
@@ -1,2 +1,133 @@
...
+- Lorem Ipsum
+
+### Celina Tjärnström
+
+- Lorem Ipsum
+
+### Tobias Hansson
+
+- Lorem Ipsum
+
```
Comment (tohanss) [inline README.md]: * Implement Git cloning of related repo

**Review by tohanss — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,134 @@
...
+Contributions to the projected is stated below.
+
+### Angelica Engström
+
+- Lorem Ipsum
+
+### Celina Tjärnström
+
+- Lorem Ipsum
```
Comment (Ceevtj) [inline README.md]:
```suggestion
- Implement functionality for compiling and testing branch
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,132 @@
...
+- [x] The team members are available for assignment to other teams.
+- [x] No further effort is being put in by the team to complete the mission.
+
+## Contributions
+
+Contributions to the projected is stated below.
+
+### Angelica Engström
+
+- Lorem Ipsum
```
Comment (angelicaengstrom) [inline README.md]: - Implement CI feature for automatic testing
- Implement output of CI features

**Review by angelicaengstrom — DISMISSED** [review]

**Pull request merged**